Parity Check Tuning/paritychecktuning.txt

```diff
@@ -20,8 +20,8 @@ Resume an array operation at=
 Send notifications for temperature related Pause or Resume=
 Parity Check Tuning debug logging=
 Requires Unraid 67 or later=Requires Unraid 6.7 or late
-ParityRead-Check History=Parity/Read-Check History=Bisherige Parity-Prüfungen
-Click to show parity-check history=Klicken um die bisherigen Parity-Prüfungen zu sehen
+ParityRead-Check History=Parity/Read-Check History=Bisherige Paritäts Prüfungen
+Click to show parity-check history=Klicke um Paritätsprüfungs Verlauf anzuzeigen
 unknown action %s=Unbekannte action: %s
```

Comment:
@SkyBird1980 Hier stimmt "action" nicht und Unbekannte würd ich klein schreiben da noch was vorangestellt wird in dem Kontext.